service/controller/key/common.go

```diff
@@ -254,30 +254,38 @@ func baseRoleARN(getter LabelsGetter, region string, accountID string, kind stri
 // CoreOS Container Linux stable 2135.4.0. AMI IDs are copied from the following
 // resource.
 //
-//     https://stable.release.core-os.net/amd64-usr/2135.4.0/coreos_production_ami_hvm.txt.
+// Current Release: CoreOS Container Linux stable 2191.5.0 (HVM)
+// AMI IDs copied from https://stable.release.core-os.net/amd64-usr/2191.5.0/coreos_production_ami_hvm.txt.
+//
+// Script to parse AMIs (Python 3):
+//
+// import urllib.request
```

Comment:
There might be a better place for this. I just wanted to keep it close to where it would need to be used.

Reply:
We are a go shop and I can't do shit in Python. Just saying. In case I should do anything here, I would probably not go that way. Having something usable for the majority of people would maybe be good. But maybe it is only me who sucks at Python.